board/safety/safety_volkswagen.h

```diff
@@ -0,0 +1,169 @@
+const int VW_MAX_STEER = 300;               // 3.0 nm
+const int VW_MAX_RT_DELTA = 130;            // max delta torque allowed for real time checks
```

Comment:
where does this come from? It should be:
`VW_MAX_RATE_UP * message_freq * VW_RT_INTERVAL / 1000000 * 1.5`

where the final extra 50% is to allow for tolerable timing delays from controlsd. what's message_freq?

Reply:
so, if it's 50Hz, then this number should be:
10 * 50 * 250000 / 1000000 * 1.5 = 188

Reply:
Send rate is 50Hz. The current value of 130 (125 rounded up) was essentially that minus the 50% pad. I've committed a change to 190 (187.5 rounded up).